packages/translation/src/lang/en.ts

```diff
@@ -677,8 +677,8 @@ export default {
         internalServerError: "Failed to fetch DNS Hole Summary",
       },
       data: {
-        adsBlockedToday: "blocked today",
-        adsBlockedTodayPercentage: "blocked today",
+        adsBlockedToday: "Blocked today",
```

Comment:
Not sure if we should write this uppercase. Because it is not on the start of the sentence

Reply:
Yeah I don't know either but the other 2 are and it looks a bit better to me.